app/code/Magento/Customer/view/frontend/email/change_email.html

```diff
@@ -8,19 +8,15 @@
 <!--@vars {
 "var store.frontend_name":"Store Name",
 "var store_email":"Store Email",
-"var store_phone":"Store Phone"
+"var store_phone":"Store Phone",
+"var customer.name":"Customer Name"
 } @-->
 {{template config_path="design/email/header_template"}}
 
-<p class="greeting">{{trans "Hello,"}}</p>
-<br>
-
+<p class="greeting">{{trans "%name," name=$customer.name}}</p>
```

Comment:
Previously we had "Hello, " which is supposed to contain something like "Hello, %name". Now it contains only "%name. Should we prefix the name with "Hello "?

Also, I'm not sure we need to translate the name. 

Please, correct me if I'm wrong

Reply:
Hi @rogyar , all other email templates use such greeting and I wanted to cleanup these to be  the same as other templates, e.g. password_new.html: `<p class="greeting">{{trans "%name," name=$customer.name}}</p>`.

If we want to drop translation for the name, then I think that this should be done in one separate PR for all emails.

Reply:
I got your point. Thank you for the clarification, that makes sense. 
Could you check the possibility to cover this change with an automated test, please?

https://github.com/magento/magento2/pull/27356#issuecomment-602177236

Thank you.

Reply:
I checked and I couldn't find similar file (like [this](https://github.com/magento/magento2/blob/6ac9924b49128fcba84f8b1168f4655da9966287/dev/tests/integration/testsuite/Magento/Email/Model/TemplateTest.php#L810)) for email template testing in Customer module however I am not very experienced with automated tests. I think this kind of email template testing is only done in Magento Email module.

Reply:
I've provided the mentioned example since it gets the template body of the specified template id. So, we can check our changes in the corresponding templates in the similar way. 

Or you can refer to the failing integration tests from one of [your PRs](https://github.com/magento/magento2/pull/27357). The integration test performs checking the email body as well. But I would recommend the [first option](https://github.com/magento/magento2/blob/6ac9924b49128fcba84f8b1168f4655da9966287/dev/tests/integration/testsuite/Magento/Email/Model/TemplateTest.php#L810)